limbo_console.gd

```diff
@@ -126,6 +128,8 @@ func hide_console() -> void:
 	if _control.visible:
 		_control.hide()
 		get_tree().paused = false
+		if _previous_gui_focus:
```

Comment:
Need to use `is_instace_valid()` because a control may be removed as an effect of a command